tls/s2n_kex.c

```diff
@@ -383,5 +244,5 @@ bool s2n_kex_includes(const struct s2n_kex *kex, const struct s2n_kex *query)
         return false;
     }
 
-    return query == kex->hybrid[0] || query == kex->hybrid[1];
+    return false;
```

Comment:
You don't need the previous kex/query check if you're just going to return false anyway